### PR TITLE
Add snippet to README pointing at useful dev tools for lens

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
cc @mrajah-twttr

File: zipkin-lens/README.md [hunk 1]
```diff
@@ -55,6 +55,22 @@ Finally, edit [App.jsx](./src/components/App/App.jsx) and
...
+the HTML does not map directly to any source code. It is recommended to install these extensions if you
+need to debug the page (they will only work with the local dev server, not a production build).
+
+### React Developer Tools
+
+Chrome: https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi
+Firefox: https://addons.mozilla.org/en-US/firefox/addon/react-devtools/
+
+### Redux DevTools
+
```
Comment (codefromthecrypt) [inline zipkin-lens/README.md:70]: this needs maybe a slightly more setup, so perhaps we can give the specific lines for lens here

**Review by codefromthecrypt — APPROVED** [review]

File: zipkin-lens/README.md (hunk 1 above)
Comment (anuraaga) [reply zipkin-lens/README.md:70]: Going to try starting up local server to see if there is actually more setup, usually there isn't so we might be missing some option when setting up the redux store.

**Comment by codefromthecrypt** [comment]
In effort to reduce the open PRs, I'm going to merge this and feel free to revise later. Thanks @anuraaga

**Pull request merged**